topics/metabolomics/tutorials/qcxms-predictions/tutorial.md

```diff
@@ -0,0 +1,594 @@
+
+![Upper panel shows the experimental spectrum of ethanol, while the lower panel shows analogical spectrum but predicted with the current workflow. The predicted peaks correspond well to the experimental ones. Intensities of simulated peaks have not been predicted perfectly, but the most important trends are preserved.](../../images/qcxms_predictions_ms_exp_pred.png "Comparison between experimental (upper panel) and predicted (lower panel) mass spectra of ethanol.")
+
+
+# Conclusion
+{% icon trophy %} Well done, you’ve simulated the mass spectrum! You might want to consult your results with the [key history](https://usegalaxy.eu/u/j.jakiela/h/ms-prediction-qcxms-single-molecule). If you would like to process multiple molecules at once, you can simply use [the workflow](https://usegalaxy.eu/u/hechth/h/end-to-end-ei-mass-spectra-prediction-workflow-using-qcxms-1) or move to “Predict MS for multiple molecules at once” tab of this tutorial to learn how the pipeline differs from the one that we’ve just covered. 
+
+
+</div>
+
+<div class="Predict-MS-for-multiple-molecules-at-once" markdown="1">
+
+## Upload data onto Galaxy
+
+We will work on two simple molecules – ethanol (C<sub>2</sub>H<sub>5</sub> OH) and ethylene (C<sub>2</sub>H<sub>4</sub>). Of course, you might add more and choose any other molecules that you want, but be aware that the more complex structure you choose, the more time it will take to complete the analysis since the workflow involves generating conformers, semiempirical methods and molecular optimisation.
+We will start with a table with the first column being molecule names and the second one – corresponding SMILES. 
+You have three options for uploading the data. The first two - importing via history and Zenodo link will give a file specific to this tutorial, while the last one – “Paste data uploader" gives you more flexibility in terms of the compounds you would like to test with this workflow. 
+
+> <hands-on-title>Option 1: Data upload - Import history</hands-on-title>
+>
+> 1. You can simply import [this history](https://usegalaxy.eu/u/j.jakiela/h/input-file-end-to-end-ei-mass-spectra-prediction-workflow-using-qcxms) with the input table. 
+>
+>    {% snippet faqs/galaxy/histories_import.md %}
+>
+> 2. **Rename** {% icon galaxy-pencil %} the history to your name of choice.
+>
+{: .hands_on}
+
+><hands-on-title>Option 2: Data upload - Add to history via Zenodo</hands-on-title>
+>
+> 1. Create a new history for this tutorial
+> 2. Import the input table from [Zenodo]({{ page.zenodo_link }})
+>
+>    ```
+>    https://zenodo.org/records/13327051/files/qcxms_prediction_input.tabular
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+{: .hands_on}
+
+> <hands-on-title> Option 3: Data Upload  - paste data </hands-on-title>
+> 
+> 1. Create a new history for this tutorial
+> 2. 
```

Comment:
This made me realize we don't have an FAQ for creating a file by pasting contents, and we probably should. Do you feel up for creating that?

Reply:
on my to-do list!